extensions/adobe/experience/offer-management/filter-rule.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/filter-rule",
+  "meta:extensible": "true",
+  "type": "object",
+  "title": "Filter Rule",
+  "description": "Filters are used to specify crirea for offer selection",
+  "definitions": {
+    "filter-rule": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the filter"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Offer name"
+        },
+        "xdm:type": {
+          "type": "string",
+          "title": "Type",
+          "description": "Filter expression type: tag rule or profile query",
+          "enum": ["tag", "profile"],
+          "meta:enum": {
+            "tag": "Tag",
```

Comment:
What is a tag?  What is a profile?  Does these relate to other objects in XDM?

Reply:
They do. We support two types of offer filtering: by the offer properties (tag rules) and by the target audience (profile query). So the `xdm:type` property here is a switch between those two.

Reply:
However, XDM already defines a Tag and a Profile.  Are these the same Tag and Profile or something else?

Reply:
This is neither Tag nor a Profile. The `type` property is a string property, which may have one of two values: either "tag" or "profile". Depending on that value, the `value` property will be processed either as a predicate on tags or as PQL query.

Reply:
I understand, @dmitry-shkolnik but using those words implies the larger context.  What about the values being _tagPredicate_ and _queryPQL_?

Reply:
@jbeckert What about renaming expression to query, having "tagQuery" and "profileQuery" constants?

Reply:
that works for me - thanks @dmitry-shkolnik

Reply:
After some deliberation I think we don't benefit much from overlaying the tag filter and profile match semantics. There is barely any place where you would use them interchangeably. We will split the schemas and remove the enum `type`.